### ISSUE TITLE
How to Normalize Center and Standardize Images With the ImageDataGenerator in Keras

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#12297]
How to Normalize, Center, and Standardize Images With the ImageDataGenerator in Keras<br>
The pixel values in images must be scaled prior to providing the images as input to a deep learning neural network model during the training or evaluation of the model.<br>
https://ift.tt/2JX2jE7
